joomla/joomla-cms
# [4.0] Remove renderfield.php from com_contact

Pull Request for Issue #34611 .

### Summary of Changes
Removed the file `components/com_contact/layouts/joomla/form/renderfield.php`
Added some parameters for the modal window.

### Testing Instructions

- Create a contact with contact form
- Activate the plugin "Content - Confirm Consent" and configure an article to be linked with.
- Go to the contact in frontend
- The field "Privacy Note" is a required field, you can't send the form without check in the box. But beside the label you will see the text "(optional)"
- Click on the link, a modal window will open
- Create a custom field of type radio for the contact form, set it as required, go to the form in frontend, you will see the text "(optional)" next to the label.


### Actual result BEFORE applying this Pull Request

![grafik](https://user-images.githubusercontent.com/9153168/123244224-fc455900-d4e3-11eb-9d7f-4d80741b06df.png)


### Expected result AFTER applying this Pull Request
No "optional" text on required fields
All fields are rendered correctly
Modal window is bigger and will match the size of the modals of the other plugins when #34537 get merged.

![grafik](https://user-images.githubusercontent.com/9153168/123244145-eb94e300-d4e3-11eb-8a93-b4a5b5b9a7d6.png)


### Documentation Changes Required
